docs/docs/getting_started.md.in

```diff
@@ -130,20 +138,18 @@ Regarding parameters for the binary, the following priority will be used:
 2. If the parameter has a value in the yaml file this value will only be used if it is not specified on the command line (second priority)
 3. If the parameter has a value as an environment variable this value will only be used if it is not specified on the command line nor yaml file
 
-Please refer to the [display backend][24] for instructions on setting display-related environment variables.
+Please refer to the [display backend][I12] for instructions on setting display-related environment variables.
 
 For specific usage instructions, refer to the README files of the respective plugins.
 
 ### Profile file format
-An example of a [_YAML_][2] [profile][23] file is
+An example of a [_YAML_][G12] [profile][G23] file is
 ```yaml
 plugins: timewarp_vk,vkdemo,ground_truth_slam,offload_vio
```

Comment:
Why putting an incomplete example here? Also, don't we need to specify something like `offload_vio.device_tx` rather than just `offload_vio`?

Reply:
fixed it up